.circleci/config.yml

```diff
@@ -26,12 +26,11 @@ jobs:
             test_with_version() {
               source "$NVM_DIR/nvm.sh"
               nvm install $1
-              nvm exec $1 $2 install
-              nvm exec $1 $2 test
+              nvm exec $1 npm install
+              nvm exec $1 npm test
             }
             case $CIRCLE_NODE_INDEX in
               0) npm install && npm test && node_modules/.bin/karma start ;;
-              1) test_with_version 4 /usr/local/bin/npm ;;
```

Comment:
I removed Node v4.x.x from the test matrix because it produces a less descriptive error message when running the [`S.encaseEither`][1] doctests. Until now we have circumvented this issue by only running doctests in Node v6.x.x.


[1]: https://sanctuary.js.org/#encaseEither